src/poliastro/twobody/orbit.py

```diff
@@ -162,9 +162,12 @@ def from_body_ephem(cls, body, epoch=None):
             warn("Input time was converted to scale='tdb' with value "
                  "{}. Use Time(..., scale='tdb') instead."
                  .format(epoch.tdb.value), TimeScaleWarning)
-
-        r, v = get_body_barycentric_posvel(body.name, epoch)
-        return cls.from_vectors(body.parent, r.xyz.to(u.km), v.xyz.to(u.km / u.day), epoch)
+        if get_velocity:
```

Comment:
Instead of hardcoding the JPL ephemeris here, the user should be responsible of setting the correct ephemeris:

```
In [3]: Orbit.from_body_ephem(Moon)
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-3-c571ef3eb3af> in <module>()
----> 1 Orbit.from_body_ephem(Moon)

~/Development/poliastro/poliastro-library/src/poliastro/twobody/orbit.py in from_body_ephem(cls, body, epoch)
    164                  .format(epoch.tdb.value), TimeScaleWarning)
    165 
--> 166         r, v = get_body_barycentric_posvel(body.name, epoch)
    167         return cls.from_vectors(body.parent, r.xyz.to(u.km), v.xyz.to(u.km / u.day), epoch)
    168 

~/.miniconda36/envs/poliastro36/lib/python3.6/site-packages/astropy/coordinates/solar_system.py in get_body_barycentric_posvel(body, time, ephemeris)
    328 
    329     """
--> 330     return _get_body_barycentric_posvel(body, time, ephemeris)
    331 
    332 

~/.miniconda36/envs/poliastro36/lib/python3.6/site-packages/astropy/coordinates/solar_system.py in _get_body_barycentric_posvel(body, time, ephemeris, get_velocity)
    223             if get_velocity:
    224                 raise KeyError("the Moon's velocity cannot be calculated with "
--> 225                                "the '{0}' ephemeris.".format(ephemeris))
    226             return calc_moon(time).cartesian
    227 

KeyError: "the Moon's velocity cannot be calculated with the 'builtin' ephemeris."

In [4]: from astropy.coordinates import solar_system_ephemeris
   ...: solar_system_ephemeris.set("jpl")
   ...: 
   ...: 
Out[4]: <ScienceState solar_system_ephemeris: 'jpl'>

In [5]: Orbit.from_body_ephem(Moon)
Out[5]: 151032073 x -151032947 km x 23.5 deg orbit around Earth (♁)
```

so I would remove this change altogether.